docs/ProductionChecklistForScalarDLLedger.md

```diff
@@ -0,0 +1,157 @@
+# Production checklist for ScalarDL Ledger
+
+This checklist provides recommendations when deploying ScalarDL Ledger in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDL Ledger on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDL Ledger
+
+The following is a checklist of recommendations when setting up ScalarDL Ledger in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardl-custom-values.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different availability zones (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDL Ledger are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Ledger pods exist on the worker nodes.
+
+In other words, the following components could run on one worker node:
```

Comment:
Does the wording for this depend on how we word a similar sentence in a different PR?: [related comment](https://github.com/scalar-labs/scalar-kubernetes/pull/213/files#r1382880277)